master/cluster.go

```diff
@@ -1227,7 +1227,7 @@ func (c *Cluster) deleteVol(name string) {
 	return
 }
 
-func (c *Cluster) markDeleteVol(name, authKey string, force bool) (err error) {
+func (c *Cluster) markDeleteVol(name, authKey string, force bool, status bool) (err error) {
```

Comment:
it's better to refactor `status` variable name.

Reply:
Changed variable name to isdelete.